.gitignore

```diff
@@ -11,3 +11,70 @@ bin
 .factorypath
 .settings
 logs
+### Scala template
+*.class
```

Comment:
Isn't `target` enough to ignore all `.class`?

Reply:
The file is generated by a plugin in IntelliJ. I never red it ^^